lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -1,4 +1,19 @@
 (function() {
+  var self = this;
+  var reporter = new window.ZendeskReporter();
```

Comment:
Will probably need a guard here, I don't think `ZendeskReporter` is available in Chat.